esp/esp/program/templatetags/intrange.py

```diff
@@ -0,0 +1,7 @@
+from django import template
+
+register = template.Library()
+
+@register.filter
+def intrange(min_val, max_val):
```

Comment:
Ew, this isn't really a filter... and having to do the calculation in the template is not great.  Is there a way to instead e.g. just have `class.grades` or something return the class's list of grades, and then we can iterate over that?